converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/Observation/ObservationParser.cs

```diff
@@ -14,8 +15,9 @@
 
 namespace Microsoft.Health.DicomCast.Core.Features.Worker.FhirTransaction;
 
-public static class ObservationParser
+internal static class ObservationParser
 {
+    [SuppressMessage("Performance", "CA1859:Use concrete types when possible for improved performance", Justification = "Preserve read-only semantics")]
```

Comment:
Curious - why suppress this error? In most other spots looks like you just updated the type

Reply:
Yeah, here is a bit trickier. I'm torn between this parse method to return an immutable type but use the concrete type. Perhaps I'll use `ImmutableList<T>` instead.